CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+0.4.0 (2020-03-17)
+------------------
+
+Features:
+
+- Add support for websockets [#20](https://github.com/IDR/idr-py/pull/20)
+
```

Comment:
Do you think we should squash the `0.4.0.dev*` entries into a single `0.4.0` entry for the final release?

Reply:
Can do